RATapi/models.py

```diff
@@ -291,6 +304,18 @@ class Layer(RATModel, populate_by_name=True):
     hydration: str = ""
     hydrate_with: Hydration = Hydration.BulkOut
 
+    @model_validator(mode="before")
+    @classmethod
+    def sld_imaginary_error(cls, data: Any):
+        """If the extra input 'sld_imaginary' is given, give a more descriptive error."""
+
+        if isinstance(data, dict) and data.get("SLD_imaginary", False):
+            raise ValueError(
+                "The standard Layer class does not support imaginary SLD. " "Use the AbsorptionLayer class instead."
```

Comment:
A personal bugbear here - could you please remove the word "standard" in this error message and the one above for the contrasts. The word "standard" has been heavily overused in RAT, so I want to remove it where possible. In particular here, "standard layers" refers to a calculation that does not use a custom function, which can have `absorption` either `True` or `False`.